package.json

```diff
@@ -28,6 +29,7 @@
     "react-redux": "^8.1.1",
     "react-router": "^6.14.1",
     "react-zoom-pan-pinch": "^3.1.0",
+    "tippy.js": "^6.3.7",
```

Comment:
You should only need the react package.

Reply:
I need the base package to import styles